store/user.js

```diff
@@ -0,0 +1,15 @@
+class User {
+  id = null
+
+  name = null
+
+  givenName = null
+
+  familyName = null
```

Comment:
We do not have these 2 fields I guess

Reply:
I took them from some example response. I will remove it.